### PR TITLE
Added support statement for Docker Swarm

### PR DESCRIPTION
**Pull request by jnorden**
Added supporting statements for Docker Swarm.  Essentially we can support it but the al-agent-container will need to be ran in a "standalone" manner on any hosts in the swarm.  The al-agent-container cannot be managed by swarm commands or through services due to the lack of support right now for --privileged mode.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ To deploy the Alert Logic Agent Container for Docker, you need your unique regis
 2. Click "Details."
 3. Copy your unique registration key.
 
-## Deploy the Agent Container
+## Deploy the Agent Container Using Standard Docker "Run" Command
```
Comment (ericreeves) [inline docker/README.md]: "Using the Standard" ?

File: docker/README.md [hunk 2]
```diff
@@ -33,5 +33,8 @@ Use the following procedure to deploy the Agent Container to a single Docker hos
 	If you use Docker version 1.12 or older, use `--cpu-period="100000"` and `--cpu-quota="300000"`.
 4. Press "Enter."
 
+## Deploy the Agent Container in Docker Swarm
+The al-agent-container requires --privileged mode in order to gain escalated access to analyize packets at the network level.  At this time, Docker Swarm does not support creating a service in "Privileged Mode".  There is an outstanding request to add this capability into Docker Swarm; however, until the open issue is resolved the recommended method is to run the al-agent-container as a standalone container on any Docker host participating in the swarm.  The al-agent-container will operate properly and will analyze the network traffic on the host it is deployed on; as well as, the container-to-container (or service-to-service) traffic.
```
Comment (ericreeves) [inline docker/README.md]: *analyze

File: docker/README.md (hunk 1 above)
Comment (jnorden) [reply docker/README.md]: yes, the standard docker run command.  When you pull swarm into the discussion there are different ways to run containers and swarm doesn't do "run".  So we'll need to separate the difference between running base docker and how it has to be done on swarm

File: docker/README.md (hunk 2 above)
Comment (ericreeves) [reply docker/README.md]: The al-agent-container will analyze network traffic to/from the host it is deployed on, as well as the container-to-container (or service-to-service) traffic.

File: docker/README.md (hunk 1 above)
Comment (ericreeves) [reply docker/README.md]: I just meant insert the word "the".  :)

Deploying the thing Using the other thing.

**Comment by jnorden** [comment]
@eric-al - fixed the "the" comment

**Review by ericreeves — DISMISSED** [review]

File: docker/README.md [hunk 3]
```diff
@@ -33,5 +33,8 @@ Use the following procedure to deploy the Agent Container to a single Docker hos
 	If you use Docker version 1.12 or older, use `--cpu-period="100000"` and `--cpu-quota="300000"`.
 4. Press "Enter."
 
+## Deploy the Agent Container in Docker Swarm
+The al-agent-container requires --privileged mode in order to gain escalated access to analyze packets at the network level.  At this time, Docker Swarm does not support creating a service in "Privileged Mode".  There is an outstanding request to add this capability into Docker Swarm; however, until the open issue is resolved the recommended method is to run the al-agent-container as a standalone container on any Docker host participating in the swarm.  The al-agent-container will operate properly and will analyze the network traffic to/from the host it is deployed on; as well as, the container-to-container (or service-to-service) traffic.
```
Comment (jnorden) [inline docker/README.md:37]: @eric-al fixed it...enjoy

**Review by ericreeves — APPROVED** [review]

**Review by jnorden — COMMENTED** [review]
all changes...done (I think)

**Pull request merged**